cpu/esp32/Makefile.features

```diff
@@ -1,14 +1,25 @@
-CPU_FAM = esp32
+ifneq (,$(filter esp32,$(CPU_FAM)))
+  CPU_ARCH = xtensa
+  CPU_CORE = xtensa-lx6
+else ifneq (,$(filter esp32c3,$(CPU_FAM)))
+  CPU_ARCH = riscv_esp32
+  CPU_CORE = riscv32
```

Comment:
There are allready some riscv32 MCU they use 
`CPU_CORE := rv32imac` (eg.: cpu/gd32v/Makefile.features)
is there a reason to have this `riscv32`

Reply:
Unfortunately, we can't use `cpu/riscv_common` as is, please see section "Background" in PR https://github.com/RIOT-OS/RIOT/pull/18260#issue-1284563955. In short
- ESP-IDF's `vector.S` and `portasm.S`, which we use, use a different context frame structure
- RISC-V based ESP32x SoCs use a special SoC register ` INTERRUPT_CORE0_CPU_INT_THRESH_REG` to enable/disable interrupts.

Reply:
Ah ok, now I got it. I must have read too superficially. You don't ask why not use `cpu/esp_common`, but why not use `rv32` as `CPU_CORE`? To be honest, I have to check whether this makes a difference.